guides/v2.1/cloud/project/project-patch.md

```diff
@@ -36,7 +36,7 @@ There are two types of patches:
 Always apply and test a patch your local system in an active branch. You can push and test in an Integration environment prior to deploying across all environments. All environments are read-only. You must update locally and push the Git branch to deploy across all environments.
 </div>
 
-Our patches are Composer driven. For more information on Composer, see [Composer in Cloud]({{ page.baseurl }}/cloud/reference/cloud-composer.html). When you perform a {{site.data.var.ece}} upgrade, you automatically upgrade with patches and hotfixes through the `composer update` command.
+Our patches are Composer driven. For more information on Composer, see [Composer in Cloud]({{ page.baseurl }}/cloud/reference/cloud-composer.html). When you perform a {{site.data.var.ece}} upgrade, you automatically upgrade with patches and hot fixes through the `composer update` command.
```

Comment:
Optional. I prefer that the "For more info" sentence be moved to the end of the paragraph.

Reply:
In the Cloud makeover, all references should be at the end of the offending paragraph in the form: "See [title of XREF](add link here)."